cypress/integration/search_spec.js

```diff
@@ -0,0 +1,130 @@
+describe('Search', function() {
+  beforeEach(function() {
+    cy.visit('/')
+    cy.get('#filter-bar').as('filterBar')
+    cy.get('#movie-list').as('movieList')
+  })
+
+  it('has "Sort By"', function() {
+    cy.get('@filterBar').contains('Sort By')
+  })
+
+  it('has input', function() {
+    cy.get('@filterBar').find('input')
+  })
+
+  it('has Search button', function() {
+    cy
+      .get('@filterBar')
+      .contains('Search')
+      .should('match', 'button')
+  })
+
+  it('has 10 default results', function() {
+    cy
+      .get('@movieList')
+      .find('.movie-row')
+      .should('have.length', 10)
```

Comment:
Again, the app data is static, but the assumption is that fixture data would predictably have enough for the default 10 rows.